assignments/function-conversion.js

```diff
@@ -1,24 +1,23 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {
-// console.log("Function was invoked!");
-// };
-// myFunction();
-
-// let anotherFunction = function (param) {
-//   return param;
-// };
-// anotherFunction("Example");
-
-// let add = function (param1, param2) {
-//   return param1 + param2;
-// };
-// add(1,2);
-
-// let subtract = function (param1, param2) {
-//   return param1 - param2;
-// };
-// subtract(1,2);
+let myFunction = () => {
+console.log("Function was invoked!");}
+
+myFunction();
+
+let anotherFunction = param => {return param}
```

Comment:
don't forget your ; also since arrow functions with a single parameter have an implicit return you do not need to have return. like this     let anotherFunction = param => param;